service/history/consts/const.go

```diff
@@ -46,10 +46,8 @@ var (
 	ErrTaskRetry = errors.New("passive task should retry due to condition in mutable state is not met")
 	// ErrDuplicate is exported temporarily for integration test
 	ErrDuplicate = errors.New("duplicate task, completing it")
-	// ErrConflict is exported temporarily for integration test
-	ErrConflict = errors.New("conditional update failed")
-	// ErrMaxAttemptsExceeded is exported temporarily for integration test
-	ErrMaxAttemptsExceeded = errors.New("maximum attempts exceeded to update history")
+	// ErrLocateCurrentWorkflowExecution is the error returned when current workflow execution can't be located
+	ErrLocateCurrentWorkflowExecution = serviceerror.NewUnavailable("unable to locate current workflow execution")
```

Comment:
why is this a retryable error?

Reply:
This only happens when workflow creates a newer run between getting current runID and loading the mutable state for that runID in `getCurrentWorkflowContext()`. So it's due to a race condition of two concurrent user requests, which can likely be resolved by a retry.